bls12_381/src/fp.rs

```diff
@@ -51,6 +61,27 @@ impl PartialEq for Fp {
     }
 }
 
+impl Ord for Fp {
+    fn cmp(&self, other: &Fp) -> Ordering {
+        // Elements are ordered lexicographically.
```

Comment:
This isn't constant time; also, although an ordering _can_ be defined for elements of a prime field, I don't think that (strictly speaking) fields should implement `(Partial)Ord`. This sounds like a question for @daira.

Reply:
I know there was something non-test-related that required the `Ord` bound, but 1fe3e3784cf9c9f1ec4da0210c6095b56fb3447a appears to only show it needed by a test. I'll try to track down where the usage was.

Reply:
Looks like it was only a test that was using this bound; I can remove it.

As a side note, `ff_derive` has implemented `Ord` and `PartialOrd` since https://github.com/zkcrypto/ff/pull/1, where I copied over the API from `pairing::bls12_381` which did at the time implement `(Partial)Ord` on fields. Should these also be removed?

Reply:
If we remove this Ord, we should remove them from derive too yeah.